backend/manager/modules/dal/src/main/java/org/ovirt/engine/core/dao/ImageTransferDao.java

```diff
@@ -35,6 +35,13 @@ public interface ImageTransferDao extends GenericDao<ImageTransfer, Guid>, Searc
      */
     List<ImageTransfer> getByStorageId(Guid storageId);
 
+    /**
+     * Retrieves ImageTransfer entities based on backup ID
```

Comment:
Can use `{@link ImageTransfer}` to make JavaDoc prettier :)

Reply:
Nice